### ISSUE TITLE
Invalid sessionToken

### ISSUE THREAD

**Issue by joostbaptist** [cedricdelpoux/react-google-places-suggest#86]
Hi! First of all thank you for building this great little component.

I've started getting this error recently, using version 3.12.1: 

```
InvalidValueError: not an instance of AutocompleteSessionToken
```

It happens when you type something, select a place, and then start typing something else (starting a second session). I suspect it has something to do with reusing the session token as implemented in #78. The problem disappears when I downgrade to version 3.11. Could it be that Google does not want us to reuse session tokens? Does anyone have any idea?

**Comment by cedricdelpoux** [comment]
I have no idea about what is the best way to use Google sessions tokens. Maybe we can add an option `withSessionToken=false` to disable it

**Comment by cedricdelpoux** [comment]
I would be happy if someone could help with a PR about this

**Comment by ItsBradyDavis** [comment]
Possibly related: https://github.com/cedricdelpoux/react-google-places-suggest/pull/97

**Comment by cedricdelpoux** [comment]
Closing it because it was related to #97.

I just published `3.12.3` version
